### ISSUE TITLE
Presenting remotely at IETF 92 Monday 3/23

### ISSUE THREAD

**Issue by sgillies** [geojson/draft-geojson#71]
We got super sort notice about https://datatracker.ietf.org/meeting/92/agenda/dispatch/ due to (as I understand it) temporary AD vacancy. I'm going to put together slides this afternoon for review here and then send them to Mary Barnes (DISPATCH WG) on Sunday.

**Comment by tschaub** [comment]
Can we tune in remotely?

Thanks for tackling this @sgillies.  Send word if any extra hands would be useful.

**Comment by sgillies** [comment]
@tschaub I'll find out and update here. The meeting is in Dallas, so the presentation is 1:30 Central.

**Comment by dret** [comment]
thanks a lot @sgillies for tackling this. if there's anything i can do to help, please let me know. but generally speaking i am sure that you're a much better fit for giving this presentation.
i don't think the AD switch is the problem. i think it's simply process to have this kind of presentation before a new WG is formed. i think the main problem is that nobody told us ahead of time.
https://datatracker.ietf.org/wg/dispatch/charter/

**Comment by dret** [comment]
in terms of remote participation, http://www.ietf.org/meeting/92/ is a good starting point to explore, and http://tools.ietf.org/agenda/92/ seems to be a good starting point for a variety of remote participation tools.

**Comment by sgillies** [comment]
Cullen Jennings, one of the Dispatch WG chairs got me up to speed on the process today. I'm feeling optimistic.

Here's the link to the audio/video/chat for sessions: http://ietf92.conf.meetecho.com/

**Comment by dret** [comment]
thanks for all the hard work, @sgillies. i'll try to attend today's session remotely.

**Comment by adoyle** [comment]
Yes, thanks!! 

I’m going to be in my car, heading for the airport so can’t tune in. I’ll send some positive brainwaves your way.

**Comment by dret** [comment]
the session is live now!

**Comment by sgillies** [comment]
GeoJSON is dispatched. Chartering to continue :)

**Comment by tschaub** [comment]
Thanks @sgillies!

Can you summarize the final bits at least (any decisions made)?  I had to mute at the end to get on another call (just when they started talking about privacy).

**Comment by sgillies** [comment]
@tschaub the job of the Dispatch WG is to take new proposals like ours and dispatch them to an IETF Area ([RAI](http://trac.tools.ietf.org/area/rai/trac/wiki) in our case) or take them out to the woods and dispatch them.

Figure of speech.

There were no objections. Area Directors are behind us, so we're going forward.

In other words, we are now at the point I thought we were at [just before Thanksgiving](https://github.com/geojson/draft-geojson/issues/58#issuecomment-63526172). But this time we really are. I'm very pleased and grateful to Cullen Jennings and Mary Barnes for their help.

**Comment by adoyle** [comment]
Thanks Sean! I looked at the chat log, it looks like it was a little chaotic.

--
